test/rest/test_rest.py

```diff
@@ -28,7 +28,7 @@ def my_function(x):
 
 def test_apply(setup):
     m = {
-        '_leaves': {
+        '_builds': {
```

Comment:
When handwriting dictionary data, retaining the _builds string instead of using KEY_BUILDS helps with readability.